cmd/proxsave/base_dir_test.go

```
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"path/filepath"
+	"sync"
+	"testing"
+)
+
+func forceDetectedBaseDirForTest(t *testing.T, baseDir string) {
+	t.Helper()
+	origInfo := execInfo
+	origOnce := execInfoOnce
```

golangci-lint (2.12.2)

[error] copylocks: assignment copies lock value to origOnce: sync.Once contains sync.noCopy

(govet)